gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -545,8 +542,26 @@ private void pollAndAdvanceDag() throws IOException, ExecutionException, Interru
       }
     }
 
-    private ExecutionStatus getJobExecutionStatus(boolean slaKilled, JobStatus jobStatus) {
-      if (slaKilled) {
+    // cancel the job and returns true if the job status remains ORCHESTRATED for some specific time
```

Comment:
Use javadoc syntax:
/**
* Cancel the job if the job has been "orphaned". A job is orphaned if {provide definition of orhphaned job}.
@return true if ....
**/